src/client.ts

```diff
@@ -211,7 +211,13 @@ export class Client {
   }
 
   /**
-   * Gets the host url prefix for current app
+   * Translates a client route like `/foo/bar` to the full URL used in the host
+   * app for the same page, e.g. `https://hostapp.com/#/client-app/foo/bar`.
+   * You should use this whenver generating an internal link within a client
+   * application so that the user gets a nice experience if they open a link in
+   * a new tab, or copy and paste a link URL into a chat message or email.
+   *
+   * @param clientRoute The /-separated path within the client app to link to.
```

Comment:
Wow, somehow I totally overlooked that this exists. Thanks!

Reply:
We have @zsdsjl to thank for the implementation! We didn't use it right away, and I forgot about it, because I have about 2k of working memory. Thanks again @zsdsjl !